frontend/src/components/configurate-podcast/configurate-podcast.tsx

```diff
@@ -35,9 +35,11 @@ const ConfiguratePodcast: React.FC = () => {
   useEffect(() => {
     if (isEdit) {
       dispatch(configuratePodcastActions.loadPodcast(Number(id)));
+    } else {
+      dispatch(configuratePodcastActions.resetState());
```

Comment:
what is the user go 2 times on edit page?)
we need to reset everting on `onMount`